src/pages/HomePage/ui/HomePage.tsx

```diff
@@ -1,6 +1,5 @@
 import { Typography } from '@mui/material';
 import Grid from '@mui/material/Unstable_Grid2';
-import AuthorizationComponent from '@widgets/authorization/Authorisation.component';
 
 const HomePageComponent = () => {
```

Comment:
Не нужно лепить длинные названия, когда и так понятно. Лучше разруливать правильно архитектурой папок и файлов.

```suggestion
const HomePage = () => {
```

У вас же TS? :)

```suggestion
const HomePageComponent: FC = () => {
```

Если есть пропсы:
```suggestion
const HomePageComponent: FC<IHomePageComponent> = () => {
```

Если пейджа:
```suggestion
const HomePageComponent: NextPage = () => {
```

Если пейджа с пропсами:
```suggestion
const HomePageComponent: NextPage<IHomePageComponent> = () => {
```

Reply:
исправил